styleguide/source/assets/scss/02-molecules/_alert.scss

```diff
@@ -0,0 +1,58 @@
+.ama__alert {
+  background-color: $homepagePurple;
+  padding: $gutter / 2;
+  color: $white;
+  width: 100%;
+  display: flex;
+  align-items: center;
+
+  @include breakpoint($bp-med min-width) {
+    background-image: url('../icons/svg/icon-homepage-alert.svg');
+    background-repeat: no-repeat;
+    background-position: 20px 50%;
+  }
+
+  @include breakpoint($bp-small max-width) {
+    flex-direction: column;
+  }
+
+  &__text {
+    font-weight: 900;
+    margin-bottom: 0;
+    margin-right: 25px;
+
+    @include breakpoint($bp-med min-width) {
+      margin-right: 0;
+      margin-left: 50px;
+      flex: 1;
+    }
+  }
+
+  button {
+    text-transform: uppercase;
+    font-weight: 900;
+    margin: 0 auto;
+
+    @include breakpoint($bp-med min-width) {
+      align-self: flex-end;
+    }
+  }
+
+  &__close {
+    height: 22px;
+    width: 22px;
+    overflow: hidden;
+    margin-left: 25px;
+
+    svg {
+      fill: $white;
+      vertical-align: inherit;
+    }
+
+    @include breakpoint($bp-small max-width) {
```

Comment:
Just a question. Why are we using max-width with our breakpoints when are are responsive first?